markdown/developers/testing_with_gitpod.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Testing with Gitpod
+subtitle: Testing, Code review and Website dev.
+---
+
+# Introduction
+
+Gitpod is an open-source developer platform that can be spun up quickly from a Git repository, containing all the programs needed to run your pipelines. It has many purposes, including: testing pipelines, learning Nextflow, reviewing/editing code and live views of edited markdown/HTML for webpage development. In addition, within Gitpod, any change can be pushed to branches for review. 
+
+Gitpod is free (with a GitHub/GitLab/Bitbucket account) for use up to 50hours/month and allows 4 parellel workspaces (as of March 2022). It is based in your browser (reconmmended with Chrome, Firefox or Edge). You can find their extensive documentation [here](https://gitpod.io/). 
+
+All nf-core repos (including the webpage) should have their own working gitpod instances, with all the code required available in each environment.
+
+## Running Gitpod on an nf-core repo
+
+First install the browser extension, follow the instructions [here](https://www.gitpod.io/docs/browser-extension). This adds the green Gitpod button to each Git repository for easy access to the environment. This button simply appends `https://gitpod.io/#` onto the git repo URL (e.g. https://gitpod.io/#https://github.com/nf-core/rnaseq).
+
+Once you click on the green button, you will asked to sign in with either Github, GitLab or Bitbucket. Once you have signed in, you should see the following:
+
+![PNG](/assets/markdown_assets/developers/gitpod/nf-core-gitpod.png)
```

Comment:
I tried to copy the other Markdown pages. Finding gitpod now displaying the images correct,, will try to find out whats going on

Reply:
OK, worked it out. You cannot use relative paths for the markdown preview. So need to change :
`![PNG](./public_html/assets/markdown_assets/developers/gitpod/nf-core-gitpod.png)`
to 
`![PNG](/public_html/assets/markdown_assets/developers/gitpod/nf-core-gitpod.png)`

Not sure if this will break the links though when rendering ?

Reply:
@ewels or @mashehu . What's the procedure for images like this?